src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+const arr = [];
+const texts = [...population].map(item => item.innerText);
+
+for (let i = 0; i < texts.length; i++) {
+  arr.push(+texts[i].split(',').join(''));
+}
+
+const sum = (prev, current) => {
+  return prev + current;
+};
+const total = arr.reduce(sum);
+const avg = Math.round(arr.reduce(sum, 0) / arr.length);
```

Comment:
```suggestion
const avg = Math.round(total / arr.length);
```
yoy already have total;